sdk/textanalytics/Azure.AI.TextAnalytics/src/SentenceSentiment.cs

```diff
@@ -17,6 +20,16 @@ internal SentenceSentiment(TextSentiment sentiment, string text, double positive
             ConfidenceScores = new SentimentConfidenceScores(positiveScore, neutralScore, negativeScore);
         }
 
+        internal SentenceSentiment(SentenceSentiment_internal sentenceSentiment)
+        {
+            // We shipped TA 5.0.0 Text == string.Empty if the service returned a null value for Text.
+            // Because we don't want to introduce a breaking change, we are transforming that null to string.Empty
+            Text = sentenceSentiment.Text ?? string.Empty;
+
+            ConfidenceScores = sentenceSentiment.ConfidenceScores;
+            Sentiment = (TextSentiment)Enum.Parse(typeof(TextSentiment), sentenceSentiment.Sentiment, ignoreCase: true);
```

Comment:
Is this enum generated from the swagger so it'll always be able to parse the latest values?

Reply:
Yes. It is originally called DocumentSentimentValue. See https://github.com/Azure/azure-sdk-for-net/blob/22fdfa6f35b2ef2d4e1204509e3ffb803ee795de/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentiment.cs#L11